api/app/controllers/api/v1/applicants_controller.rb

```diff
@@ -34,6 +39,17 @@ def update
                 render_error(applicant.errors)
             end
         end
+
+        # /applicants/delete or /applicants/:id/delete
```

Comment:
`/applicants/delete` can't happen right? Because you require the id.

Reply:
Actually, it can thanks to the `config/routes.rb`. Though, I have no idea why there are two ways to delete. @siefkenj was the one who set up this structure. You would have to ask him if you want to know why it was setup this way.

Reply:
I think I'll delete the former. It's kind of redundant anyways.

Reply:
Nevermind, I just realized the `/applicants/delete` exist because without it, this route is taken as `/applicants/:id`, where `id=delete`...

Reply:
I deleted the `/applicants/delete` sort of route and kept only the `/applicants/:id/delete` type of routes.